### ISSUE TITLE
Scaling of cohort crown area in the output files?

### ISSUE THREAD

**Issue by mpaiao** [NGEET/fates#1126]
I am adding a new crown area variable to the output (by size class and PFT). I was trying to build on existing variables in FatesHistoryInterfaceMod.F90, but I am somewhat confused. When defining crown area by size class, the cohort variable is scaled by `1 / m2_per_ha`:

https://github.com/NGEET/fates/blob/5551a70019013c2e73f63c09f3afd786d8151908/main/FatesHistoryInterfaceMod.F90#L3386-L3387

https://github.com/NGEET/fates/blob/5551a70019013c2e73f63c09f3afd786d8151908/main/FatesHistoryInterfaceMod.F90#L3535-L3536

However, when defining crown area by PFT, the cohort variable is scaled by `AREA_INV`:

https://github.com/NGEET/fates/blob/5551a70019013c2e73f63c09f3afd786d8151908/main/FatesHistoryInterfaceMod.F90#L3041-L3048

Is there any reason for the different scaling, or should both be scaled by `AREA_INV`?

**Comment by ckoven** [comment]
@mpaiao I think it is the same conversion? i.e. multiplying by `AREA_INV` is the same as dividing by `m2_per_ha`.  I think the implied meaning of the unit conversion is very slightly different though; `AREA` and `AREA_INV` are what set the nominal size of a FATES site to be 1 hectare, but in theory at least that could vary, and if it did, then the `m2_per_ha` would no longer be the correct unit conversion. So in principle `AREA_INV` would seem to be a very slightly safer unit conversion. Also seems weird that the two constants live in different modules.

https://github.com/NGEET/fates/blob/5551a70019013c2e73f63c09f3afd786d8151908/main/EDTypesMod.F90#L76-L77

https://github.com/NGEET/fates/blob/5551a70019013c2e73f63c09f3afd786d8151908/main/FatesConstantsMod.F90#L223-L224

**Comment by mpaiao** [comment]
@ckoven Thanks for clarifying. In the short term, I will replace `m2_per_ha` with `AREA_INV` when I submit the PR for the additional crown area variable, if anything, to be consistent with the other crown area conversions.

But maybe we can have a long-term discussion on whether or not to even have a nominal area at some point. If nplant is defined as a true demographic density (stems m<sup>&minus;2</sup>), then the site area is not really needed for anything, and it would reduce the number of multiplication operations in the code.

**Comment by ckoven** [comment]
@mpaiao agreed, that would be nice (but also will probably touch everything!)
